src/mscorlib/src/System/ThrowHelper.cs

```diff
@@ -416,7 +416,8 @@ internal enum ExceptionArgument
         type,
         stateMachine,
         pHandle,
-        values
+        values,
+        task
```

Comment:
You'll likely need to augment the ThrowHelper.cs in corert as well; I put ValueTask in the shared partition, but ThrowHelper isn't there.